docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.8"
+services:
+  mote:
+    build: .
```

Comment:
After `build`, you can add the following line:

```
command: python3 runmote.py
```

in `docker-compose.yml`

Reply:
Note to myself: `ADD command: python3 runmote.py at L#5`